src/ass.js

```diff
@@ -56,13 +56,9 @@ var ASS = (function (m, fs, util, path, mustache)
 
 	ass.render = function(callback)
 	{
-		fs.readFile(path.resolve(__dirname, 'templates/ass.mustache'), 'UTF-8', function(err, result){
-			if(err) {
-				return callback(err);
-			}
-			callback(null, mustache.render(result, _structured));
-		});
+		const result = fs.readFileSync(path.resolve(__dirname, 'templates/ass.mustache'), 'UTF-8');
```

Comment:
thanks for the contribution,
shouldn't there be a try/catch to catch any errors that may happen while reading the file?

Reply:
You're completely right !
Please see the latest version of my branch for the update.